client/src/static/catalog.js

```diff
@@ -1,4 +1,8 @@
 
+export const APP_LINK = "https://avonehr.com";
+
+export const APP_LOGIN_LINK = "https://app.avonehr.com";
```

Comment:
Can we use environment variable for production and development?

Reply:
Yes, sure.